### PR TITLE
Release 0.2.0

### PR DESCRIPTION
**Pull request by martinRenou**
Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by martinRenou** [comment]
@conda-forge-admin, please rerender

**Comment by martinRenou** [comment]
Oh we are in a circular dependency mess here, we need JupyterLab as a host dependency (for building the labextension we are using `jlpm`, but jupyterlab depends on nbconvert -> depends on jupyterlab_pygments. 

I am going to patch this release here, and upstream the patch for 0.2.1.

**Comment by martinRenou** [comment]
> I am going to patch this release here, and upstream the patch for 0.2.1.

Not as simple as that actually. We don't only need `jlpm` but the `jupyter labextension` Python scripts.

**Comment by martinRenou** [comment]
@conda-forge-admin, please rerender

**Comment by martinRenou** [comment]
Let's continue with this PR and open an issue for that.

**Comment by martinRenou** [comment]
As said by @jtpio, we actually don't really need JupyterLab as host dependency nor Yarn because the PyPi sdist package ships the labextension builts assets, so it should be fine.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -16,15 +16,19 @@ build:
...
   run:
-    - python
+    - python >=3.7
     - pygments >=2.4.1,<3
 
 test:
   imports:
     - jupyterlab_pygments
+  commands:
+    - test -f ${PREFIX}/share/jupyter/labextensions/jupyterlab_pygments/package.json                            # [unix]
```
Comment (jtpio) [inline recipe/meta.yaml:30]: Yes this is already a good way to test whether the extension prebuilt assets are in the archive :+1:

**Pull request merged**